keyboards/keychron/q5/readme.md

```diff
@@ -0,0 +1,19 @@
+# Keychron Q5
+
+A customizable 96% keyboard.
+
+* Keyboard Maintainer: [Keychron](https://github.com/keychron)
+* Hardware Supported: Keychron Q5
+* Hardware Availability: [Keychron](https://www.keychron.com)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make keychron/q5/rev_0150:default
+
+Flashing example for this keyboard:
+
+    make keychron/q5/rev_0150:default:flash
+
+**Reset Key**: Hold down the key located at *K00*, commonly programmed as *Esc* while plugging in the keyboard.
```

Comment:
Probably want to change this to maybe describe what each revision is, and to direct users to the revision readme for more info.

Reply:
Thanks, i will arrange it.